lib/chef/resource/windows_certificate.rb

```diff
@@ -230,13 +252,14 @@ def export_cert(cert_obj, cert_path)
           out_file.close
         end
 
-        def cert_location
-          @location ||= new_resource.user_store ? "CurrentUser" : "LocalMachine"
+        def get_cert_location
+          new_resource.user_store ? "CurrentUser" : "LocalMachine"
+          # @location ||= new_resource.user_store ? "CurrentUser" : "LocalMachine"
```

Comment:
it seems fine to memo-ify this, why are we un-memo-ifying it?

Reply:
I couldn't find anywhere in the code that was calling @location so I commented it out for now.